python_scripts/extension_documentation.py

```diff
@@ -89,23 +103,19 @@ def print_hooks():
 
     """
     i = 1
-    with open("{}/WikispeechHooks.php".format(extension_path)) as f:
+    with open("{}/WikispeechHooks.php".format(args.extension_path)) as f:
         for line in f:
             match = RE_HOOK_FUNCTION.match(line)
             if match:
-                print("|hook{} = {}".format(i, match.group(1)))
+                print("| hook{} = {}".format(i, match.group(1)))
                 i += 1
 
 
 if __name__ == "__main__":
-    if len(sys.argv) != 2:
-        print("Extension directory must be supplied.")
-        sys.exit(1)
-    extension_path = sys.argv[1]
-    extension_json_path = "{}/extension.json".format(extension_path)
-    try:
-        extension_json_file = open(extension_json_path)
-    except:
-        print("Failed to open file: {}".format(extension_json_path))
-        sys.exit(1)
-    print_extension_json_info(extension_json_file)
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--extension-path", "-e", default=".")
+    parser.add_argument("--manifest-version", "-m", default="2", type=int)
```

Comment:
can this not be decoded from the `manifest_version` key ?

Reply:
It sure can. Fixed that.